task/prefetch-dependencies-oci-ta/0.1/prefetch-dependencies-oci-ta.yaml

```diff
@@ -114,7 +114,7 @@ spec:
         name: workdir
   steps:
     - name: skip-ta
-      image: registry.access.redhat.com/ubi9/ubi-minimal:9.4-1227.1726694542@sha256:f5d2c6a1e0c86e4234ea601552dbabb4ced0e013a1efcbfb439f1f6a7a9275b0
+      image: registry.access.redhat.com/ubi9/ubi-minimal:9.5-1731604394@sha256:ab4eac109595d4595b6a978cd41c6310d7eec36122ae593a14448edfbfd7c297
```

Comment:
@zregvart is there some way to make `ta-generator` respect Renovate updates? Currently, the older image is hardcoded in recipe.yaml